lib/haskell/natural4/test/testdata/golden/xpile/simala/rodents-and-vermin.expected

```diff
@@ -0,0 +1,2 @@
+Failed transpilation:
+Unsupported RelationalPredicate: RPis
```

Comment:
Is this the excpected result?

Reply:
Yes! This is intended as documenting the current capabilties of the transpilation.
I struggled to identify all possible meanings of predicates. In this case, 
```
"Loss or Damage" IS ANY ( "caused by rodents"
                                    , "caused by insects"
                                    , "caused by vermin"
                                    , "caused by birds"
                                    )
```

Is a weird RelationalPredicate in my opinion. `ANY` feels overloaded. It may mean `OR`, but `IS ANY` suddenly means `ELEM`, and there is actually a `RPelem` `RelationalPredicate` which we don't use here.

So, I didn't even try to support this example, but rather we should talk about, which composite operators we want to support, or whether we rather want to use some `ELEM` operator.

Reply:
Good point! I pretty much have forgotten that RPelem existed 😁

Reply:
Yeah in general I think we shouldn't be afraid to try to make the syntax more regimented and understandable (not least because there is a non-trivial chance we'll actually have to try to help other people learn and use this!)